core/codenvy-hosted-platform-api-impl/src/main/java/com/codenvy/api/workspace/LimitsCheckingWorkspaceManager.java

```diff
@@ -111,6 +114,14 @@ public WorkspaceImpl startWorkspace(String workspaceId,
                                                                            ServerException,
                                                                            ConflictException {
         final WorkspaceImpl workspace = getWorkspace(workspaceId);
+        try {
+            userManager.getByName(workspace.getNamespace());
+        } catch (NotFoundException e) {
+            throw new ServerException(String.format(
```

Comment:
Are you sure that we need this check? then why we don't need it here https://github.com/codenvy/codenvy/pull/319/files#diff-8b6273ced2f7db76f99f8877a8ae652cR138